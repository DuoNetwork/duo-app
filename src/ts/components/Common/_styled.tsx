import { Menu, Modal } from 'antd';
import styled, { injectGlobal } from 'styled-components';
import { ColorStyles } from '../../common/styles';

export const SMenu = styled(Menu)`
	background: ${ColorStyles.CardBackgroundSolid};
	padding: 0;
	border-radius: 0;
`;

export const SItem = styled(Menu.Item)`
	display: flex;
	flex-direction: row;
	align-items: center;
	color: ${ColorStyles.TextWhiteAlphaLL};
	font-size: 10px;
	font-weight: 400;
	font-family: 'Roboto', 'Microsoft YaHei';
	padding: 5px 10px;
	.picker-box {
		width: 12px;
		height: 12px;
		border: 1px solid;
		border-color: ${ColorStyles.BorderWhite4};
		margin-right: 5px;
	}
	&:hover {
		background: ${ColorStyles.HoverBackgroundSolid};
	}
`;

export const SModal = styled(Modal as any)`
	color: ${ColorStyles.ThemeTextAlpha};
	.ant-modal-content {
		border-radius: 0;
		background: ${ColorStyles.CardBackgroundSolid};
	}
	.ant-modal-header {
		border-radius: 0;
		background: transparent;
		border-bottom: 1px solid rgba(0, 0, 0, 0.1);
	}
	.ant-modal-footer {
		border-radius: 0;
		background: transparent;
		border-top: 1px solid rgba(0, 0, 0, 0.1);
	}
	.ant-modal-title {
		color: ${ColorStyles.ThemeTextAlpha};
	}
	.ant-modal-close {
		color: ${ColorStyles.ThemeTextAlphaL};
	}
	.ant-modal-close:focus,
	.ant-modal-close:hover {
		color: ${ColorStyles.ThemeTextAlpha};
	}
`;

injectGlobal([
	`
	body {
		.ant-dropdown-menu-item-selected {
			color: ${ColorStyles.TextWhiteAlpha};
			background: ${ColorStyles.SelectBackgroundSolid};
			font-weight: 500;
		}
		.check-icon {
			display: block;
			color: ${ColorStyles.TextWhiteAlphaL};
		}
		.card-source-button {
			margin-left: 10px;
			width: 80px;
			height: 24px;
			border: 1px solid;
			border-color: ${ColorStyles.BorderWhite1};
			font-size: 10px;
			color: ${ColorStyles.TextWhiteAlphaL};
			display: flex;
			flex-direction: row;
			justify-content: space-between;
			align-items: center;
			padding: 0 5px;
			letter-spacing: 0;
			font-weight: 400;
		}
		.ant-dropdown-trigger .anticon-down {
			color: ${ColorStyles.TextWhiteAlphaLLL};
		}
		.ant-btn {
			border-radius: 0;
			line-height: 14px;
		}
		.ant-btn:before {
			background: ${ColorStyles.BorderBlack2};
		}
		.ant-btn-ghost {
			background-color: transparent;
			color: ${ColorStyles.ThemeTextAlpha};
			border: 1px solid;
			border-color: ${ColorStyles.BorderBlack2};
			transition: color 0.2s ease-in-out, border-color 0.2s ease-in-out,
				background-color 0.2s ease-in-out;
		}
		.ant-btn-ghost:hover, .ant-btn-ghost:focus {
			color: ${ColorStyles.ThemeText};
			border-color: ${ColorStyles.BorderBlack4};
			background-color: ${ColorStyles.LightHoverSolid};
		}
		.ledger-account-selector > .ant-radio-wrapper {
			color: ${ColorStyles.ThemeTextAlpha};
			width: 100%;
			padding: 2px 0;
		}
		.ledger-account-selector > .ant-radio-wrapper:nth-child(odd) {
			background: transparent;
		}
		.ledger-account-selector > .ant-radio-wrapper:hover {
			background: ${ColorStyles.LightHoverSolid};
		}
		.ledger-account-selector > .ant-radio-wrapper-checked {
			background: ${ColorStyles.SelectBackgroundSolid} !important;
			box-shadow:inset 0px 0px 0px 1px ${ColorStyles.BorderBlack3};
		}
		.locale-select {
			display: flex;
			align-items: center;
			margin-left: 10px;
			.ant-select-selection-selected-value {
				width: 72px;
			}
		}
		.locale-img {
			width: 14px;
			height: 14px;
			margin-right: 2px;
		}
	}
`
] as any);
