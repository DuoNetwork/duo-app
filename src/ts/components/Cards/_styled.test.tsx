// fix for @ledgerhq/hw-transport-u2f 4.28.0
import '@babel/polyfill';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import * as React from 'react';
import * as Styled from './_styled';

describe('Styled Test', () => {
	it('CSS', () => {
		const SCard = shallow(<Styled.SCard />);
		expect(SCard).toMatchSnapshot();

		const SCardTitle = shallow(<Styled.SCardTitle />);
		expect(SCardTitle).toMatchSnapshot();

		const SCardTitleSelector = shallow(<Styled.SCardTitleSelector />);
		expect(SCardTitleSelector).toMatchSnapshot();

		const SCardPriceTag = shallow(<Styled.SCardPriceTag />);
		expect(SCardPriceTag).toMatchSnapshot();

		const SCardAssetTag = shallow(<Styled.SCardAssetTag />);
		expect(SCardAssetTag).toMatchSnapshot();

		const SCardExtraDiv = shallow(<Styled.SCardExtraDiv />);
		expect(SCardExtraDiv).toMatchSnapshot();

		const SCardExtraDivSolid = shallow(<Styled.SCardExtraDivSolid />);
		expect(SCardExtraDivSolid).toMatchSnapshot();

		const SCardExtendExtraDiv = shallow(<Styled.SCardExtendExtraDiv />);
		expect(SCardExtendExtraDiv).toMatchSnapshot();

		const SCardRadioExtraDiv = shallow(<Styled.SCardRadioExtraDiv />);
		expect(SCardRadioExtraDiv).toMatchSnapshot();

		const SRadioGroup = shallow(<Styled.SRadioGroup />);
		expect(SRadioGroup).toMatchSnapshot();

		const SCardList = shallow(<Styled.SCardList />);
		expect(SCardList).toMatchSnapshot();

		const SCardListProgressBar = shallow(<Styled.SCardListProgressBar index={10} total={10} />);
		expect(SCardListProgressBar).toMatchSnapshot();

		const SCardConversionForm = shallow(<Styled.SCardConversionForm />);
		expect(SCardConversionForm).toMatchSnapshot();

		const SCardTransactionForm = shallow(<Styled.SCardTransactionForm />);
		expect(SCardTransactionForm).toMatchSnapshot();

		const SInput = shallow(<Styled.SInput />);
		expect(SInput).toMatchSnapshot();

		const STableWrapper = shallow(<Styled.STableWrapper />);
		expect(STableWrapper).toMatchSnapshot();

		const SRefreshButton = shallow(<Styled.SRefreshButton />);
		expect(SRefreshButton).toMatchSnapshot();
	});
});
