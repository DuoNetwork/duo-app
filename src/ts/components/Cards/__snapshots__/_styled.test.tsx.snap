// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Test CSS 1`] = `
.c0 {
  overflow: hidden;
  max-width: 1060px;
  background: rgba(255,255,255,1);
  box-shadow: 0 2px 5px 0 rgba(0,0,0,0.2);
  border-radius: 4px;
  border: none !important;
}

.c0 > .ant-card-head {
  height: 50px;
  background: rgba(0,114,206,1) !important;
  background: none;
  border-bottom: none;
  padding: 0 16px;
}

.c0 > .ant-card-body {
  padding: 0px 16px 16px 16px;
}

<Card
  className="c0"
/>
`;

exports[`Styled Test CSS 2`] = `
.c0 {
  color: rgba(255,255,255,1);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  font-size: 16px;
  padding-left: 12px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 3`] = `
.c0 .ant-select-selection {
  border-radius: 0;
  color: rgba(255,255,255,.8);
  background-color: transparent;
  border: 1px solid;
  border-color: rgba(255,255,255,.3);
  font-size: 10px;
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 400;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
}

.c0 .ant-select-selection span {
  color: rgba(255,255,255,.6);
}

.c0 .ant-select-selection:focus {
  -webkit-box-shadow: 0 0 2px 2px rgba(255,255,255,0.1);
  box-shadow: 0 0 2px 2px rgba(255,255,255,0.1);
}

<Select
  choiceTransitionName="zoom"
  className="c0"
  showSearch={false}
  transitionName="slide-up"
/>
`;

exports[`Styled Test CSS 4`] = `
.c0 {
  height: 100px;
  width: 165px;
  position: relative;
  margin-top: 10px;
  border: 1px dashed;
  border-color: rgba(64,79,84,.3);
  overflow: hidden;
  padding-top: 10px;
}

.c0 .bg-logo {
  height: 100px;
  width: 100px;
  position: absolute;
  right: -35px;
  top: 10px;
}

.c0 .bg-logo > img {
  height: 100%;
  width: 100%;
  opacity: 0.05;
  pointer-events: none;
}

.c0 .tag-title {
  width: 90px;
  margin-left: 20px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 .tag-title img {
  width: 10px;
  height: 10px;
  margin-left: 6px;
  opacity: 0.6;
}

.c0 .tag-title > a {
  margin: 0;
}

.c0 .tag-title > a,
.c0 .tag-title > h3 {
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 12px;
  color: rgba(64,79,84,.8);
  margin: 0;
  z-index: 99;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 .tag-content {
  width: 120px;
  margin-left: 20px;
  margin-top: 10px;
}

.c0 .tag-content .tag-subtext {
  font-size: 12px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  color: rgba(64,79,84,.8);
}

.c0 .tag-price {
  color: rgba(64,79,84,1);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 16px;
  margin-bottom: 5px;
}

.c0 .tag-unit {
  margin-left: 2px;
  color: rgba(255,255,255,.6);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 10px;
  margin-top: 5.5px;
}

.c0 .tag-price-1 {
  color: rgba(105,188,255,1);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 16px;
  margin-bottom: 5px;
}

.c0 .tag-price-2 {
  color: rgba(249,174,50,1);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 16px;
  margin-bottom: 5px;
}

.c0 .tag-price-3 {
  color: white;
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 12px;
  margin-bottom: 5px;
  margin-left: 6px;
  width: 60px;
  text-align: right;
}

.c0 .tag-unit-1,
.c0 .tag-unit-2 {
  margin-left: 2px;
  color: rgba(255,255,255,.6);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 10px;
  margin-top: 5.5px;
}

.c0 .tag-unit-3 {
  color: rgba(255,255,255,.6);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 10px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 5`] = `
.c0 {
  height: 100px;
  width: 165px;
  position: relative;
  margin-top: 10px;
  border: 1px dashed;
  border-color: rgba(64,79,84,.3);
  overflow: hidden;
  padding-top: 10px;
}

.c0 .bg-logo {
  height: 100px;
  width: 100px;
  position: absolute;
  right: -35px;
  top: 10px;
}

.c0 .bg-logo > img {
  height: 100%;
  width: 100%;
  opacity: 0.05;
  pointer-events: none;
}

.c0 .tag-title {
  width: 90px;
  margin-left: 15px;
}

.c0 .tag-title > h3 {
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 12px;
  color: rgba(64,79,84,.8);
  margin: 0;
}

.c0 .tag-content {
  width: 140px;
  margin-left: 15px;
  margin-top: 10px;
}

.c0 .tag-subtext {
  color: rgba(64,79,84,.8);
  font-size: 10px;
}

.c0 .tag-price {
  color: rgba(64,79,84,1);
  font-family: 'Roboto','Microsoft YaHei';
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 18px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 6`] = `
.c0 {
  color: rgba(255,255,255,.8);
  font-size: 10px;
  padding-right: 10px;
  line-height: 24px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 7`] = `
.c0 span {
  color: rgba(255,255,255,.6);
  font-size: 12px;
  padding-right: 10px;
  line-height: 24px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 span img {
  margin-left: 4px;
  width: 10px;
  height: 10px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 8`] = `
.c0 {
  font-family: 'Roboto','Microsoft YaHei';
  color: rgba(255,255,255,.8);
  font-size: 10px;
  padding-right: 10px;
  line-height: 24px;
}

.c0 > .extend-extra-wrapper {
  padding: 0 10px;
  border: 1px dashed;
  border-color: rgba(255,255,255,.3);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c0 > .extend-extra-wrapper .tag-content {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: rgba(255,255,255,.6);
  overflow: hidden;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row-reverse;
  -ms-flex-direction: row-reverse;
  flex-direction: row-reverse;
  margin-left: 8px;
  max-width: 42px;
  -webkit-transition: max-width 0.3s ease-in-out,margin-left 0.3s ease-in-out;
  transition: max-width 0.3s ease-in-out,margin-left 0.3s ease-in-out;
  -webkit-transition: max-width 0.3s ease-in-out,margin-left 0.3s ease-in-out;
}

.c0 > .extend-extra-wrapper:hover > .tag-content {
  margin-left: 8px;
  max-width: 300px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 9`] = `
.c0 {
  font-family: 'Roboto','Microsoft YaHei';
  color: rgba(255,255,255,.8);
  font-size: 10px;
  line-height: 24px;
}

.c0 > .extend-extra-wrapper {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c0 > .true {
  padding-right: 10px;
}

.c0 .ant-radio-group {
  font-size: 10px;
  font-weight: 400;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 10`] = `
.c0 {
  margin-left: 8px;
}

.c0 .ant-radio-button-wrapper:first-child,
.c0 .ant-radio-button-wrapper:last-child {
  border-radius: 0;
}

.c0 .ant-radio-button-wrapper:last-child {
  border-left: 0;
}

.c0 .ant-radio-button-wrapper {
  padding: 0 12px;
  border: 1px dashed;
  border-color: rgba(255,255,255,.2);
  color: rgba(255,255,255,.2);
  background: transparent;
}

.c0 .ant-radio-button-wrapper-checked {
  border: 1px solid;
  border-color: rgba(255,255,255,1);
  color: rgba(255,255,255,.8);
  background: rgba(255,255,255,.1);
  box-shadow: -1px 0 0 0 rgba(255,255,255,.6);
}

<RadioGroup
  buttonStyle="outline"
  className="c0"
  disabled={false}
/>
`;

exports[`Styled Test CSS 11`] = `
.c0 {
  width: 100%;
}

.c0 .status-list-wrapper {
  width: 100%;
}

.c0 .status-list-wrapper > ul:last-child {
  margin: 10px 0;
}

.c0 .status-list-wrapper > ul {
  list-style: none;
  margin: 10px 0;
  padding: 10px 5px;
  border: 1px dashed;
  border-color: rgba(64,79,84,.3);
}

.c0 .status-list-wrapper > ul li:nth-child(even) {
  background-color: rgba(255,255,255,.04);
}

.c0 .status-list-wrapper > ul .no-bg {
  background-color: transparent !important;
  padding: 0 5px 0 3px;
  overflow: hidden;
  position: relative;
}

.c0 .status-list-wrapper > ul .block-title {
  font-weight: 600;
  color: rgba(64,79,84,1);
  margin-bottom: 5px;
}

.c0 .status-list-wrapper > ul .block-title .last-reset-title {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 .status-list-wrapper > ul .block-title .last-reset-title .last-reset-title-span {
  color: rgba(64,79,84,.8);
  font-size: 10px;
}

.c0 .status-list-wrapper > ul .t-center {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0 .status-list-wrapper > ul .no-borderbottom {
  border-bottom: none !important;
}

.c0 .status-list-wrapper > ul li {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0 5px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-bottom: 1px solid rgba(64,79,84,.1);
}

.c0 .status-list-wrapper > ul li .title {
  color: rgba(64,79,84,.8);
}

.c0 .status-list-wrapper > ul li .content {
  color: rgba(64,79,84,1);
}

.c0 .status-list-wrapper > ul li .percent-button {
  outline: none;
  cursor: pointer;
  font-family: 'Roboto','Microsoft YaHei';
  background-color: transparent;
  color: rgba(64,79,84,.8);
  border: 1px dashed;
  border-color: rgba(64,79,84,.2);
  font-size: 10px;
  height: 24px;
  width: 40px;
  -webkit-transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out;
  transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out;
}

.c0 .status-list-wrapper > ul li .p_mobile {
  width: 22% !important;
  font-size: 9px !important;
}

.c0 .status-list-wrapper > ul li .percent-button:hover {
  color: rgba(64,79,84,1);
  border-color: rgba(64,79,84,.4);
  background-color: rgba(255,255,255,0.1);
}

.c0 .status-list-wrapper > ul li .align-right {
  width: 100%;
  color: rgba(64,79,84,1);
  font-size: 12px;
  text-align: right;
}

.c0 .status-list-wrapper > ul li .default-button {
  cursor: pointer;
  font-size: 10px;
  color: rgba(64,79,84,1);
  border: 1px dashed;
  border-color: rgba(64,79,84,.2);
  padding: 0 5px;
  -webkit-transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out;
  transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out;
}

.c0 .status-list-wrapper > ul li .default-button:hover {
  color: rgba(64,79,84,1);
  border-color: rgba(64,79,84,.4);
}

.c0 .status-list-wrapper > ul li:last-child {
  border-bottom: none;
}

.c0 .status-list-wrapper > ul .input-line {
  padding: 5px 5px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 5px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
}

.c0 .status-list-wrapper > ul .input-disabled {
  pointer-events: none;
  opacity: 0.2 !important;
}

.c0 .status-list-wrapper > ul .description {
  width: 100%;
  height: 21px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: rgba(64,79,84,.8);
  margin-bottom: 5px;
  font-size: 12px;
}

.c0 .status-list-wrapper > ul .description img {
  width: 10px;
  height: 10px;
  opacity: 0.6;
}

.c0 .status-list-wrapper > ul .img-line {
  width: 100%;
  height: 108px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 .status-list-wrapper > ul .img-line .demo-img {
  height: 90%;
  opacity: 0.6;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 12`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c0 .bar-bg {
  width: 285px;
  height: 12px;
  background: rgba(255,255,255,1);
  box-shadow: inset 0 1px 6px 0px rgba(0,0,0,0.6);
  border-radius: 6px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-left: 3px;
  padding-right: 3px;
}

.c0 .bar-bg .inner-bar {
  position: relative;
  width: 279px;
  height: 6px;
  background: rgba(255,255,255,.4);
  box-shadow: 0 1px 3px 1px rgba(0,0,0,0.2);
  border-radius: 3px;
}

.c0 .bar-bg .inner-bar:after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
  background-image: -webkit-gradient( linear, 0 0, 100% 100%, color-stop(0.25,rgba(0,0,0,0.15)), color-stop(0.25,transparent), color-stop(0.5,transparent), color-stop(0.5,rgba(0,0,0,0.15)), color-stop(0.75,rgba(0,0,0,0.15)), color-stop(0.75,transparent), to(transparent) );
  z-index: 1;
  background-size: 60px 60px;
  -webkit-animation: move 3s linear infinite;
  animation: move 3s linear infinite;
  overflow: hidden;
}

.c0 .bar-text {
  color: rgba(255,255,255,.8);
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 13`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 .conv-button {
  outline: none;
  cursor: pointer;
  width: 180px;
  font-family: 'Roboto','Microsoft YaHei';
  background-color: transparent;
  color: rgba(64,79,84,.8);
  border: 1px dashed;
  border-color: rgba(64,79,84,.2);
  -webkit-transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
  transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
}

.c0 .mobile {
  width: 46% !important;
}

.c0 .conv-button:hover {
  color: rgba(64,79,84,1);
  border-color: rgba(64,79,84,.3);
  background-color: #e6f7ff;
}

.c0 .selected {
  height: 22px;
  border: 1px solid !important;
  border-color: rgba(64,79,84,.4) !important;
  color: rgba(64,79,84,1) !important;
  background-color: 'transparent' !important;
}

.c0 .non-select {
  height: 22px;
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 14`] = `
.c0 {
  width: 100%;
  margin-top: -10px;
}

.c0 .token-button {
  outline: none;
  cursor: pointer;
  width: 80px;
  font-family: 'Roboto','Microsoft YaHei';
  font-size: 10px;
  background-color: transparent;
  color: rgba(64,79,84,.8);
  border: 1px dashed;
  border-color: rgba(64,79,84,.3);
  -webkit-transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
  transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
}

.c0 .token-button:hover {
  color: rgba(64,79,84,1);
  border-color: rgba(64,79,84,.3);
  background-color: #e6f7ff;
}

.c0 .trans-button {
  outline: none;
  cursor: pointer;
  width: 115px;
  font-family: 'Roboto','Microsoft YaHei';
  background-color: transparent;
  color: rgba(64,79,84,.8);
  border: 1px dashed;
  border-color: rgba(64,79,84,.3);
  -webkit-transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
  transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
}

.c0 .trans-button .superscript {
  margin-left: 2px;
  font-size: 12px;
}

.c0 .trans-button:hover {
  color: rgba(64,79,84,1);
  border-color: rgba(64,79,84,.3);
  background-color: #e6f7ff;
}

.c0 .selected {
  border: 1px solid !important;
  border-color: rgba(64,79,84,.3) !important;
  color: rgba(64,79,84,1) !important;
  background-color: 'transparent' !important;
}

.c0 .wide {
  width: 175px !important;
}

.c0 .form-button {
  outline: none;
  cursor: pointer;
  width: 120px;
  font-family: 'Roboto','Microsoft YaHei';
  background-color: transparent;
  color: rgba(64,79,84,1);
  border: 1px solid;
  border-color: rgba(64,79,84,.3);
  -webkit-transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
  transition: color 0.2s ease-in-out,border-color 0.2s ease-in-out, background-color 0.2s ease-in-out;
}

.c0 .b_mobile {
  width: 30% !important;
}

.c0 .form-button:hover {
  color: rgba(64,79,84,1);
  border-color: rgba(64,79,84,.4);
  background-color: #e6f7ff;
}

.c0 .form-button:disabled {
  pointer-events: none;
  opacity: 0.2 !important;
}

.c0 .remark {
  margin-top: 20px;
  font-size: 12px;
  padding: 0 10px;
  text-align: justify;
  color: rgba(64,79,84,.6);
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 15`] = `
.c0 {
  outline: none;
  font-size: 12px;
  background: rgba(255,255,255,1);
  color: rgba(64,79,84,1);
  border: 1px solid rgba(64,79,84,.3);
  width: 160px;
  height: 28px;
  padding: 0 8px;
  text-align: left;
  -webkit-transition: box-shadow 0.3s;
  transition: box-shadow 0.3s;
}

.c0:focus {
  box-shadow: inset 0 1px 6px 0px rgba(0,0,0,0.1),0 0 2px 2px rgba(255,255,255,0.05);
}

.c0::-webkit-input-placeholder {
  color: rgba(64,79,84,.6);
}

.c0::-moz-placeholder {
  color: rgba(64,79,84,.6);
}

.c0:-ms-input-placeholder {
  color: rgba(64,79,84,.6);
}

.c0::placeholder {
  color: rgba(64,79,84,.6);
}

.c0.input-error {
  box-shadow: inset 0 1px 6px 0px rgba(0,0,0,0.1),0 0 2px 2px rgba(255,86,86,0.2) !important;
}

.c0:disabled {
  cursor: not-allowed;
}

<input
  className="c0"
/>
`;

exports[`Styled Test CSS 16`] = `
.c0 {
  margin-top: 10px;
}

.c0 thead > tr > th {
  background: 'transparent';
  color: rgba(64,79,84,.8);
  border-bottom: 0;
}

.c0 tbody > tr:nth-child(even) > td {
  background: 'transparent';
}

.c0 tbody > tr:nth-child(odd) > td {
  background: transparent;
}

.c0 tbody > tr:hover > td {
  background: rgba(25,112,197,1);
}

.c0 td {
  cursor: pointer;
  border-bottom: 0 !important;
  color: rgba(64,79,84,.8);
}

.c0 tr > .eth,
.c0 tr > .token-ab,
.c0 tr > .fee {
  text-align: right;
}

.c0 tbody > tr > .fee {
  color: rgb(245,83,83);
}

.c0 .Redeem .eth {
  color: rgb(97,206,94);
}

.c0 .Redeem .token-ab {
  color: rgb(245,83,83);
}

.c0 .Create .token-ab {
  color: rgb(97,206,94);
}

.c0 .Create .eth {
  color: rgb(245,83,83);
}

.c0 .ant-table table {
  border: none;
  border-radius: 0;
}

.c0 .ant-table-placeholder {
  width: 100%;
  background: none !important;
  border-bottom: 1px solid;
  border-left: 1px solid;
  border-right: 1px solid;
  border-color: rgba(64,79,84,.1);
  border-radius: 0;
  color: rgba(64,79,84,1);
}

<div
  className="c0"
/>
`;

exports[`Styled Test CSS 17`] = `
.c0 {
  color: rgba(255,255,255,.4);
  border: none !important;
  background: transparent !important;
}

.c0:hover {
  color: rgba(255,255,255,.8);
}

.c0:focus {
  color: rgba(255,255,255,.6);
}

.c0:after {
  border: 0 solid rgba(255,255,255,.5);
}

.c0:disabled {
  color: rgba(255,255,255,.2);
}

<Button
  block={false}
  className="c0"
  ghost={false}
  loading={false}
/>
`;
